docs/whatisk8s.md

```diff
@@ -0,0 +1,51 @@
+# What is Kubernetes?
+
+The Kubernetes project was started by Google in 2014. 
+
+Kubernetes is an open-source platform for automating deployment, scaling, and operations of application containers across clusters of hosts. Kubernetes builds upon a [decade and a half of experience that Google has with running production workloads at scale](https://research.google.com/pubs/pub43438.html), combined with best-of-breed ideas and practices from the community.
```

Comment:
I don't think condensing this text into a single paragraph is an improvement. It's harder to parse.